vmdb/app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -35,4 +35,14 @@
           %li
             = link_to("#{ui_lookup(:table => "container_node")}: #{@record.container_node.name}",
                 {:controller => 'container_node', :action => 'show', :id => @record.container_node},
-                :title => _("View %s %s") % [ui_lookup(:table => "container_node"), @record.container_node.name])
\ No newline at end of file
+                :title => _("View %s %s") % [ui_lookup(:table => "container_node"), @record.container_node.name])
+        - if role_allows(:feature => "container_service_show_list")
+          - num_services = @record.number_of(:container_services)
+          - if num_services == 0
+            %li.disabled
+              = link_to(_("Container Services (%s)") % num_services, "#")
```

Comment:
@zeari can you change thsi and the line below to use ui_lookup call to be consistent, currently labels on summary screen and listnav do not match, Label on summary screen says Services: and label on listnav says Container Services: